pkg/networkverifier/networkverifier.go

```diff
@@ -13,9 +13,10 @@ import (
 	"github.com/openshift/configuration-anomaly-detection/pkg/logging"
 
 	hivev1 "github.com/openshift/hive/apis/hive/v1"
-	"github.com/openshift/osd-network-verifier/pkg/helpers"
+	"github.com/openshift/osd-network-verifier/pkg/data/cloud"
+	"github.com/openshift/osd-network-verifier/pkg/data/cpu"
+	"github.com/openshift/osd-network-verifier/pkg/probes/curl"
 	"github.com/openshift/osd-network-verifier/pkg/proxy"
-	"github.com/openshift/osd-network-verifier/pkg/verifier"
 	onv "github.com/openshift/osd-network-verifier/pkg/verifier"
```

Comment:
I appreciate you fixing this duplicate import, but perhaps we can keep the un-renamed version instead of the alias? Some people feel strongly about not using "ONV" 😅

Reply:
Yeah good call, I agree, I tend to dislike package aliases :)